docs/basics/101-139-s3.rst

```diff
@@ -0,0 +1,314 @@
+   extracts files from the annexed archives, etc. Don’t envision a special-remote
+   as merely a physical place or location – a special-remote is a protocol that
+   defines the underlying transport of your files to and/or from a specific location.
+
+In this section, we provide a walkthrough on how to set up Amazon S3 for hosting
+your DataLad dataset, and how to access this data locally from GitHub.
+
+Prerequisites
+^^^^^^^^^^^^^
+In order to use Amazon S3 for hosting your datasets, you need to:
+
+- Signup for an `AWS account <https://aws.amazon.com/>`_
+- Verify your account
+- Find your AWS access key
+- Signup for a `GitHub account <https://github.com/join>`_
+- Optional: install the `AWS Command Line Interface <https://aws.amazon.com/cli/>`_
+
+The `AWS signup <https://aws.amazon.com/>`_ procedure requires you to provide your
+e-mail address, physical address, and credit card details before verification is possible. 
+
+To find your AWS access key, log in to the `AWS Console <https://console.aws.amazon.com/>`_,
+open the dropdown menu at your username (top right), and select "My Security
+Credentials". A new page will open with several options, including "Access keys
+(access key ID and secret access key)" from where you can select "Create New Access
+Key" or access existing credentials. Take note to copy both the "Access Key ID" and
+"Secret Access Key".
+
+.. figure:: ../artwork/src/aws_s3_create_access_key.png
+
+   Create a new AWS access key from "My Security Credentials"
+
+To ensure that your access key details are known when initializing the special
+remote, export them in your shell:
+
+.. code-block:: bash
+
+   $ export AWS_ACCESS_KEY_ID="<your-access-key-ID>"
+   $ export AWS_SECRET_ACCESS_KEY"<your-secret-access-key>"
+
+In order to work directly with AWS via your command-line shell, you can
+`install the AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html>`_.
+However, that is not required for this walkthrough.
+
+Lastly, to publish your data repository to GitHub, from which users will be able do install
+the complete dataset, you will need a `GitHub account <https://github.com/join>`_.
+
+Your DataLad dataset
+^^^^^^^^^^^^^^^^^^^^
+For this walkthrough, we are using a basic sample neuroimaging dataset with
+anatomical and functional images from a single subject as well as some
+metadata:
+
+.. code-block:: bash
+
+   $ cd /tmp/sample_neuro_data
+   $ tree
+   .
+   ├── dataset_description.json
+   ├── participants.tsv
+   ├── sub-01
+   │   ├── anat
+   │   │   └── sub-01_T1w.nii.gz
+   │   └── func
+   │       ├── sub-01_task-rest_run-1_bold.nii.gz
+   │       └── sub-01_task-rest_run-2_bold.nii.gz
+   └── task-rest_bold.json
+
+The first step is to ensure that you have a valid DataLad dataset,
+with ``main`` as the default branch.
+
+.. importantnote:: Ensure main is set as default branch for newly-created repositories
```

Comment:
#722 adds an FAQ on this that you could link to using
```
 :ref:`some random text of your choice <gitannexdefault>`
```

Reply:
Great, thanks!